test/Sema/immutability.swift

```diff
@@ -430,7 +430,25 @@ func QoI() {
   get_only = 92            // expected-error {{cannot assign to value: 'get_only' is a get-only property}}
 }
 
+func func1() -> Int { return 7 }
 
+func func2() {
+    func func3() {}
```

Comment:
One last nit - could you please format this to be 2 spaces like the rest of the file?

Reply:
My bad...One sec